modules/external-services/rds.tf

```diff
@@ -54,6 +54,7 @@ resource "aws_rds_cluster" "tfe" {
   preferred_backup_window   = "07:00-09:00"
   vpc_security_group_ids    = [aws_security_group.db_access.id]
   final_snapshot_identifier = "${var.prefix}tfe-${var.install_id}-final"
+  skip_final_snapshot       = var.demo == true ? true : false
```

Comment:
Technically speaking, you don't need the ternary here since the variable is defined as a boolean type already. Just an FYI, nothing merge blocking.

Reply:
Yeah, at first I thought it looked better to be super explicit, but looking at this this morning I agree it can go. Thanks! Removed. :+1: